src/components/dialogs/NewEntryDialog.tsx

```
@@ -0,0 +1,791 @@
+                      }`}
+                    />
+                    <span>{option.name}</span>
+                  </SelectItem>
+                ))}
+              </SelectContent>
+            </Select>
+          </div>
+        );
+
+      case NOTION_PROPERTY_TYPES.STATUS:
+        const selectedStatus = property.status?.options?.find((opt: any) => opt.name === value);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)